plugins/woocommerce/tests/performance/README.md

```diff
@@ -57,8 +57,8 @@ Before using the tests a test environment is needed to run against.
 We first spin up an environment using `wp-env` and configure that environment with the necessary plugins and data using the Initialization Script [`init-sample-products.sh`](https://github.com/woocommerce/woocommerce/tree/trunk/plugins/woocommerce/tests/performance/bin/init-sample-products.sh) that will set up a shop with sample products imported and the shop settings (payment method, permalinks, address etc) needed for the tests already set. It is recommended using this to just see the tests in action.
 
 ```sh
-pnpm run env:dev --filter=woocommerce
-pnpm run env:performance-init --filter=woocommerce
+pnpm run env:dev --filter=@woocommerce/plugin-woocommerce
```

Comment:
This command is loading e2e-environment instead of wp-env.

Reply:
That script runs `"env:dev": "pnpm wp-env start --update"` and should be setting up `wp-env`.

Reply:
> That script runs "env:dev": "pnpm wp-env start --update" and should be setting up wp-env.

`run` needs to be removed.

```suggestion
pnpm env:dev --filter=@woocommerce/plugin-woocommerce
```